src/Kernel/RectorKernel.php

```diff
@@ -4,62 +4,64 @@
 
 namespace Rector\Core\Kernel;
 
-use Rector\Core\Config\Loader\ConfigureCallMergingLoaderFactory;
-use Rector\Core\DependencyInjection\Collector\ConfigureCallValuesCollector;
-use Rector\Core\DependencyInjection\CompilerPass\AutowireArrayParameterCompilerPass;
-use Rector\Core\DependencyInjection\CompilerPass\AutowireRectorCompilerPass;
-use Rector\Core\DependencyInjection\CompilerPass\MakeRectorsPublicCompilerPass;
-use Rector\Core\DependencyInjection\CompilerPass\MergeImportedRectorConfigureCallValuesCompilerPass;
-use Rector\Core\DependencyInjection\CompilerPass\RemoveSkippedRectorsCompilerPass;
 use Rector\Core\Exception\ShouldNotHappenException;
-use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Rector\Core\Util\FileHasher;
+use Rector\Testing\PHPUnit\StaticPHPUnitEnvironment;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\DependencyInjection\ContainerInterface;
+use Webmozart\Assert\Assert;
 
 final class RectorKernel
 {
-    private readonly ConfigureCallValuesCollector $configureCallValuesCollector;
+    /**
+     * @var string
+     */
+    private const CACHE_KEY = 'v7';
```

Comment:
should it be updated when add new param to `__construct` ? see error https://github.com/rectorphp/rector-src/actions/runs/4982917436/jobs/8919255393#step:6:26

```

There was 1 error:

1) Rector\Core\Tests\Issues\DowngradeNewInArrayInitializerPromotion\DowngradeNewInArrayInitializerPromotionTest::test with data set #0
ArgumentCountError: Too few arguments to function Rector\Core\PhpParser\Parser\InlineCodeParser::__construct(), 3 passed in /tmp/rector/kernel-v7-b923df1ea94cad9fb5b38891cbfaa566ed8e88d7e8551c1c3c978857f94fc876.php on line 3246 and exactly 4 expected
```